tests/de/cover_HassTurnOff.yaml

```diff
@@ -14,6 +14,16 @@ tests:
         name: Rollo links
         domain: cover
     response: geschlossen
+  - sentences:
+      - schließ den Vorhang links
+      - Vorhang links schliessen
+      - mach den Vorhang links herunter
```

Comment:
tbh, this sounds weird (_this counts also for all similar others_)

Reply:
I agree, I just did not want to make the RegEx too complicated - better some more cases than too little cases was my line of thought here. Anyways, thanks for merging!